aip/auth/4111.md

```diff
@@ -0,0 +1,84 @@
+--- 
+aip: 
+  id: 4111
+  scope: auth
+  state: approved
+  created: 2018-07-21
+  updated: 2019-07-21
+permalink: /auth/4111
+redirect_from:
+  - /4111
+---
+
+# Self-signed JWT-based Access Token
+
+JWT access token for Google APIs was introduced in 2014 to provide a more
+efficient auth stack for Cloud APIs being accessed via service accounts by
+bypassing the intermediate step of exchanging client assertions for OAuth
+tokens. It becomes a standard authentication method, such that many Cloud APIs
```

Comment:
The sentence "It becomes..." feels a bit run-on. How about we define Self-signed JWT first:
A self-signed JWT is a spec-conforming JWT that is locally signed by the service account private key. Since many Cloud APIs accept self-signed JWT in place of traditional access tokens, it is now considered a standard authentication method.